after.Mono.Cecil.sln.targets

```diff
@@ -0,0 +1,5 @@
+<Project>
```

Comment:
Could you explain the relation between this and Directory.Build.targets? I'm a bit confused at to what those do.

Reply:
This is a tips to fix the dotnet test on a solution so that the non tests are not executed and failing https://github.com/Microsoft/vstest/issues/1129#issuecomment-344762968